src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/LspLogger.cs

```diff
@@ -37,7 +37,7 @@ public IDisposable BeginScope<TState>(TState state)
 
     public bool IsEnabled(LogLevel logLevel)
     {
-        return true;
+        return logLevel >= _logLevel;
```

Comment:
Turns out we were just logging everything in VS Code regardless of log level set. This fixes that and makes our logs a lot quieter by default

Reply:
Reverting this, will log an issue to follow up properly

```suggestion
        return true;
```